src/components/utils/test/CMakeLists.txt

```diff
@@ -112,4 +127,13 @@ create_test("utils_test" "${testSources}" "${testLibraries}")
 
 add_subdirectory(test_generator)
 
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /FORCE" )
+  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /IGNORE:4099" )
```

Comment:
@OHerasym 
Can you add some comment on error code? It will help to understand why it has been ignored.